app/views/RoomInfoEditView/index.tsx

```diff
@@ -347,21 +353,16 @@ class RoomInfoEditView extends React.Component<IRoomInfoEditViewProps, IRoomInfo
 		try {
 			const db = database.active;
 			const subCollection = db.get('subscriptions');
-			const teamChannels = await subCollection.query(
-				Q.where('team_id', room.teamId as string),
-				Q.where('team_main', Q.notEq(true))
-			);
+			const teamChannels = await subCollection
+				.query(Q.where('team_id', room.teamId as string), Q.where('team_main', Q.notEq(true)))
+				.fetch();
```

Comment:
I understand that adding fetch here is correct, but how did it work before without fetch?

Reply:
I took a look at the doc and did some tests and it works both ways.